### DISCUSSION TITLE
GPIO shutdown

### DISCUSSION THREAD

**Discussion by RuslanKADR** [tesla-android/issue-tracker#321] [Ideas]
Hi, Mike. 

I want to install a UPS HAT on the raspberry pi 4 a supercapacitor to properly turn off when the tesla car falls asleep. How can you organize a software shutdown of the device via GPIO. Here a link of device:

https://sl.aliexpress.ru/p?key=OwAYslX

**Comment by herrfrei** [comment]
Sounds good to me! Hopefully it can be integrated.
